pkg/controller/seed-controller-manager/kubernetes/resources.go

```diff
@@ -325,9 +324,6 @@ func GetDeploymentCreators(data *resources.TemplateData, enableAPIserverOIDCAuth
 		)
 	}
 
-	if data.Cluster().Annotations[kubermaticv1.AnnotationNameClusterAutoscalerEnabled] != "" {
```

Comment:
Does the constant then still need to exist?

Reply:
I don't think so, good point. Let me remove it.